UnityProject/Assets/Scripts/HealthV2/Living/CirculatorySystem/CirculatorySystemBase.cs

```diff
@@ -204,6 +205,30 @@ public void BloodSaturationCalculations(float HeartEfficiency)
 			{
 				foreach (var KVP in bloodAndValues.Value)
 				{
+					var PurityMultiplier = 1f;
+
+					var BloodPressure = 1f;
+
+					var PercentageBloodPressure = BloodPool.Total  / StartingBlood;
+					if (PercentageBloodPressure < 0.75f)
+					{
+						BloodPressure = PercentageBloodPressure / 0.75f;
+					}
+
+					if (PercentageBloodPressure > 1.25f)
+					{
+						healthMaster.ChangeBleedStacks(1); //TODO Change to per body part instead
+
+					}
+
+
+					var Percentage =  BloodPool.GetPercent(bloodAndValues.Key);
+
+					if (Percentage < 0.33f)
+					{
+						PurityMultiplier = Percentage / 0.33f;
```

Comment:
yeah, I don't get it. Where did you even get the 075 and 0.33 from? Make them a well named variable at least.

Reply:
Just arbitrary s balancing numbers